exercises/sheet4/Alberto_Ander_Filip/Sheet4.tex

```diff
@@ -0,0 +1,944 @@
+	Without loss of generality we may assume that the $c^i$'s such that
+	$\bm{c^i}(v)>0$ are $\bm{c^1},\dots, \bm{c^j}$.
+	Given $s\in P, t\in \RR$ satisfying $s + tv \in P$, 
+	there exists an $\eps > 0$ such that 
+	$s + (t+ \eps)v\in P$ 
+	if and only if 
+	$\bm{c^i}(s+tv) < c^i_0$ 
+	for $i\in \{1,\dots, j\}$.\par %this is a simple computation
+	Then $\max$ in the definition of $f_v$ can be computed with
+	only $j$ inequalities:
+	\[ 
+	\max \{t\in \RR \, : \, s + tv\in P\} =
+	\max \left\{ t \in \RR \, : \, \bm{c^i}(s+tv) \leq c^i_0 \ \ \forall i \in \{1,\dots, j\}\right\} 
+	\]
+	Since we are computing a maximum restricted to inequalities,
+	it can be computed as
+	\[
+	f_v(s) = \min_{1\leq i \leq j} 
+	\Big\{\max \{ t \in \RR \, : \, \bm{c^i}(s+tv)\leq c^i_0\}\Big\}.
+	\]
+	For all $i\in\{1,\dots,j\}$
+	it is satisfied that 
+	$\max \{ t \in \RR \, : \, \bm{c^i}(s+tv)\leq c^i_0\} =
+	\frac{c^i_0-\bm{c^i}(s)}{\bm{c^i}(v)}$ for $1\leq i \leq j$. 
+	In consequence 
+	$f_v(s)=\min_{1\leq i \leq j} \frac{c^i_0-\bm{c^i}(s)}{\bm{c^i}(v)}$.
+	Then $f_v$ is continuous since it is the minimum of
+	finitely many continuous functions.
+\end{proof}
+
+We have the following corollary.
+
+\begin{corollary}
+	\label{lem:lemilla}
+	Given $s\in  {P}$ and $v\in\RR^p$ such that $s+v\in   P$.
+	There exists $U\subseteq   P$, 
+	an open set (with the topology of $  P$), $s\in U$,
+	and $\eps > 0$ such that $U+\eps v \subseteq   P$.
+\end{corollary}
+
+\begin{lemma} \label{lemm:dimm}
+    $\dim \Sigma(P,Q) \geq \dim P - \dim Q$
+\end{lemma} 
+\begin{proof}
+	Notice that the interior of P is non-empty, 
+	as $\dim P=p$.	
+	Let $u\in int \, P$ and $r=\pi(u)$. 
+	Then the fiber $\hat{P}\defs \pi^{-1}(r)$ 
+	is a polytope of dimension $p-q$. 
+	Indeed, 
+	$\hat{P}= P \cap (u + \ker \overline\pi)
+	\supset B \cap (u + \ker \overline\pi)$ 
+	for some open ball
+	$B\subset P$ centered in $u$. 
+	Trivially 
+	$\dim (B \cap (u + \ker \overline\pi)) = 
+	\dim \ker \overline\pi=p-q$.\par
+	Now take $\gamma$ a section. 
+	By definition $\gamma(r)\in \hat{P}$. 
+	As $\hat{P}$ is a polytope with direction $\ker \overline\pi$,
+	there exist linearly independent vectors 
+	$v_1,\dots, v_{p-q}\in \ker \overline\pi$ such that 
+	$\gamma(r)+ v_i\in \hat{P}$ and 
+	in particular $\gamma(r)+ v_i \in P$ for every $i$.
+	Let us denote $s\defs \frac{1}{\vol(Q)}\int_Q\gamma(x) \,\mathbb{d}x$. 
+	By definition $s \in \Sigma(P,Q)$. 
+	We will show that there exist $\delta_1,\dots, \delta_{p-q} > 0$ such that $s + \delta_i v_i \in \Sigma(P,Q)$ for each $i$. \par  
+	Fix $1\leq i \leq p-q$. 
+	Using $\cref{lem:lemilla}$ we conclude that 
+	there exists an open (relative to $P$) set $U_i$ containing $s$ and $\eps_i>0$ 
+	such that $z+ \eps_i v_i \in P$ for every $z\in U_i$.  
+	Let us define $V_i \defs \gamma^{-1}(U_i)$.
+	This set is open because $\gamma$ is continuous and $U_i$ is open.
+	Then we define the function $f_i \colon \RR^q \to \RR$
```

Comment:
You're not defining f_i, you're just asking for one that satisfies some requirements!